ethcore/vm/src/ext.rs

```diff
@@ -136,8 +139,11 @@ pub trait Ext {
 	/// then A depth is 0, B is 1, C is 2 and so on.
 	fn depth(&self) -> usize;
 
-	/// Increments sstore refunds count by 1.
-	fn inc_sstore_clears(&mut self);
+	/// Increments sstore refunds counter.
+	fn inc_sstore_refund(&mut self, value: U256);
```

Comment:
I'd rather call it `add` and `sub` instead of `inc` and `dec`, since it takes a parameter now. (or change the comments to say "Increase/Decrease" instead of "Increment/Decrement")

Reply:
Fixed!